packages/parser/parse/normalize.js

```diff
@@ -102,6 +102,7 @@ export default function normalizeValue(token) {
         offsetY: normalizeValue({ $type: 'dimension', $value: layer.offsetY ?? 0 }),
         blur: normalizeValue({ $type: 'dimension', $value: layer.blur ?? 0 }),
         spread: normalizeValue({ $type: 'dimension', $value: layer.spread ?? 0 }),
+        inset: layer.inset === true || undefined,
```

Comment:
If it doesn’t change anything, I think we should just force this to always be a boolean. That just makes things simpler down-the-line. It will default to `false`. We do the same thing with numbers/dimensions—defaulting to `0` if not specified.

```suggestion
        inset: layer.inset === true,
```